src/buoyancy.jl

```diff
@@ -79,17 +97,17 @@ salinity or absolute salinity, where applicable.
 #####
 
 """
-    LinearEquationOfState{T} <: AbstractEquationOfState
+    LinearEquationOfState{FT} <: AbstractEquationOfState
 
 Linear equation of state for seawater. 
 """
-struct LinearEquationOfState{T} <: AbstractEquationOfState
-    α :: T
-    β :: T
+struct LinearEquationOfState{FT} <: AbstractEquationOfState
+    α :: FT
+    β :: FT
```

Comment:
Not for this PR but I agree with your suggestion on Slack that we should use something more descriptive like `thermal_expansion` and `haline_contraction`.

Reply:
Ok, we will get to that.